tgui/packages/tgui/interfaces/Techweb.js

```diff
@@ -766,6 +767,14 @@ const TechNode = (props, context) => {
               Research
             </Button>
           )}
+          {
+            ((node_tier > tech_tier+1)) && (
+              <Tooltip
+                content={"Researching this costs discovery points inefficiently. Please research tier "+(tier+1)+" techs first."}>
```

Comment:
`content={"Researching this node will cost additional discovery points. Please research more tier "+(tier+1)+" technology nodes first."}>`

Reply:
thanks lol